<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectByAttributes</class>
 <widget class="QMainWindow" name="SelectByAttributes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="mainGridLayout">
      <item row="1" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="cancelPushButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="applyPushButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QGroupBox" name="inputGroupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="inputGroupBoxGridLayout" rowstretch="0,0,0,4,4,0,2,0,0" columnstretch="3,2,1">
           <item row="2" column="1">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2">
            <widget class="QToolButton" name="inFeatureToolButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QGroupBox" name="uniqueValuesGroupBox">
             <property name="title">
              <string>Unique values</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QGridLayout" name="uniqueValuesGroupBoxGridLayout">
                <item row="0" column="0">
                 <widget class="QListView" name="uniqueListView"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QListView" name="fieldsListView"/>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QComboBox" name="inFeatureComboBox"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="inFeatureLabel">
             <property name="text">
              <string>Input feature dataset</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QLineEdit" name="outFeatureLineEdit"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Fields</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Output feature</string>
             </property>
            </widget>
           </item>
           <item row="8" column="2">
            <widget class="QToolButton" name="outFeatureToolButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Operators</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QGridLayout" name="operatorsGroupBoxGridLayout">
                <item row="2" column="1">
                 <widget class="QToolButton" name="less_equalToolButton">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>&lt;=</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="3">
                 <widget class="QToolButton" name="inToolButton">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>In</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QToolButton" name="notToolButton">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Not</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QToolButton" name="isToolButton">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Is</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QToolButton" name="likeToolButton">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Like</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QToolButton" name="orToolButton">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Or</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QToolButton" name="greater_equalToolButton">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>&gt;=</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QToolButton" name="equalToolButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>=</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QToolButton" name="greaterToolButton">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>&gt;</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QToolButton" name="lessToolButton">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>&lt;</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QToolButton" name="andToolButton">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>And</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QToolButton" name="nullToolButton">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Null</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="getUniqueValuesPushButton">
             <property name="text">
              <string>Get Field Unique Values</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="3">
            <widget class="QTextEdit" name="textEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>inFeatureComboBox</tabstop>
  <tabstop>inFeatureToolButton</tabstop>
  <tabstop>fieldsListView</tabstop>
  <tabstop>equalToolButton</tabstop>
  <tabstop>nullToolButton</tabstop>
  <tabstop>likeToolButton</tabstop>
  <tabstop>notToolButton</tabstop>
  <tabstop>greaterToolButton</tabstop>
  <tabstop>greater_equalToolButton</tabstop>
  <tabstop>andToolButton</tabstop>
  <tabstop>isToolButton</tabstop>
  <tabstop>lessToolButton</tabstop>
  <tabstop>less_equalToolButton</tabstop>
  <tabstop>orToolButton</tabstop>
  <tabstop>inToolButton</tabstop>
  <tabstop>uniqueListView</tabstop>
  <tabstop>getUniqueValuesPushButton</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>outFeatureLineEdit</tabstop>
  <tabstop>outFeatureToolButton</tabstop>
  <tabstop>cancelPushButton</tabstop>
  <tabstop>applyPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
